systemd/google-cloud-ops-agent.service

```diff
@@ -14,7 +14,8 @@
 
 [Unit]
 Description=Google Cloud Ops Agent
-Wants=google-cloud-ops-agent-fluent-bit.service google-cloud-ops-agent-opentelemetry-collector.service google-cloud-ops-agent-diagnostics.service
+Wants=google-cloud-ops-agent-fluent-bit.service google-cloud-ops-agent-opentelemetry-collector.service google-cloud-ops-agent-diagnostics.service network-online.target
+After=network-online.target
```

Comment:
The subagents also set `Requires=network-online.target`. Any reason we don't need that here? 

Also, can we remove the dependency from the subagents since that dep is added here and the subagents depends on this service?

Reply:
I'd prefer to leave the subagent dependency there since they depend on it for their own reasons. Also helps reduce the number of things we need to change if we ever move the startup checks into a different service.

Reply:
Fair enough, I agree